osu.Framework/Graphics/UserInterface/TextBox.cs

```diff
@@ -798,11 +767,9 @@ protected override bool OnKeyDown(KeyDownEvent e)
                     return false;
             }
 
-            return base.OnKeyDown(e) || consumingText;
+            return (textInput?.Active ?? false) || base.OnKeyDown(e);
```

Comment:
would prefer to write
```suggestion
            return (textInput?.Active == true) || base.OnKeyDown(e);
```